Patches/JDS EFT Apparel/Apparel_Armor.xml

```diff
@@ -0,0 +1,336 @@
+          <value>
+            <ArmorRating_Sharp>15</ArmorRating_Sharp>
+          </value>
+        </li>
+
+        <li Class="PatchOperationReplace">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_Highcom_Trooper_TFO_Armor" or
+              defName = "JDSTarkov_6B13_Assault_Armor" or
+              defName = "JDSTarkov_6B23-2_Armor"
+            ]/statBases/ArmorRating_Blunt
+          </xpath>
+          <value>
+            <ArmorRating_Blunt>26</ArmorRating_Blunt>
+          </value>
+        </li>
+
+        <li Class="PatchOperationReplace">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_Highcom_Trooper_TFO_Armor" or
+              defName = "JDSTarkov_6B13_Assault_Armor" or
+              defName = "JDSTarkov_6B23-2_Armor"
+            ]/statBases/ArmorRating_Heat
+          </xpath>
+          <value>
+            <ArmorRating_Heat>0.36</ArmorRating_Heat>
+          </value>
+        </li>
+
+        <!-- === Class 5 === -->
+
+        <!-- == FORT Redut-M / 6B13 M / IOTV Gen4 / BNTI Gzhel-K / FORT Defender-2 == -->
+         <li Class="PatchOperationAdd">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_FORT_Redut-M_Body_Armor" or
+              defName = "JDSTarkov_6B13_M_Assault_Armor" or
+              defName = "JDSTarkov_IOTV_Gen4_Armor" or
+              defName = "JDSTarkov_BNTI_Gzhel-K_Armor" or
+              defName = "JDSTarkov_FORT_Defender-2_Body_Armor"
+            ]/statBases
+          </xpath>
+          <value>
+            <Bulk>7</Bulk>
+            <WornBulk>5</WornBulk>
+          </value>
+        </li>
+
+        <!-- == IOTV Gen4 Armor (Assault) / IOTV Gen4 Armor (Assault) / FORT Redut-T5 == -->
+        <li Class="PatchOperationAdd">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_IOTV_Gen4_ArmorAssaultkit" or
+              defName = "JDSTarkov_IOTV_Gen4_ArmorFullProtection" or
+              defName = "JDSTarkov_FORT_Redut-T5_Body_Armor"
+            ]/statBases
+          </xpath>
+          <value>
+            <Bulk>8.5</Bulk>
+            <WornBulk>3.5</WornBulk>
+          </value>
+        </li>
+
+        <!-- == statBases == -->
+        <li Class="PatchOperationReplace">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_FORT_Redut-M_Body_Armor" or
+              defName = "JDSTarkov_6B13_M_Assault_Armor" or
+              defName = "JDSTarkov_IOTV_Gen4_Armor" or
+              defName = "JDSTarkov_BNTI_Gzhel-K_Armor" or
+              defName = "JDSTarkov_FORT_Defender-2_Body_Armor" or
+              defName = "JDSTarkov_IOTV_Gen4_ArmorAssaultkit" or
+              defName = "JDSTarkov_IOTV_Gen4_ArmorFullProtection" or
+              defName = "JDSTarkov_FORT_Redut-T5_Body_Armor"
+            ]/statBases/ArmorRating_Sharp
+          </xpath>
+          <value>
+            <ArmorRating_Sharp>13</ArmorRating_Sharp>
```

Comment:
I'd kick this up to 14 or maybe 14.5 to make this tier equivalent to composite armor, or perhaps a touch better since it's perhaps somewhat heavier armor.